app/index/controller.js

```diff
@@ -144,7 +144,7 @@ export default Controller.extend({
         return total.add(moment().diff(current.get("from")));
       }, moment.duration());
 
-    this.set("_activeActivityDuration", duration);
+    // this.set("_activeActivityDuration", duration);
```

Comment:
Is this safe to just comment out?

Reply:
Well spotted! I will check!

Reply:
>  ERROR: Error: Assertion Failed: You attempted to update `_activeActivityDuration` on `<timed@controller:index::ember9070>`, but it had already been used previously in the same computation.  Attempting to update a value after using it in a computation can cause logical errors, infinite revalidation bugs, and performance issues, and is not supported.

Ah :thinking: that's why he did it! 
The code in question:

```js
  /**
   * The duration sum of all activities of the selected day
   *
   * @property {moment.duration} activitySum
   * @public
   */
  activitySum: computed(
    "_activities.@each.{fromTime,duration}",
    "_activeActivityDuration",
    function() {
      this._activitySum();

      return this.get("_activities")
        .rejectBy("active")
        .reduce((total, current) => {
          return total.add(current.get("duration"));
        }, this.get("_activeActivityDuration"));
    }
  ),

  /**
   * Compute the current activity sum
   *
   * @method _activitySum
   * @private
   */
  _activitySum() {
    const duration = this.get("_activities")
      .filterBy("active")
      .reduce((total, current) => {
        return total.add(moment().diff(current.get("from")));
      }, moment.duration());

    this.set("_activeActivityDuration", duration);
  },
```

Commenting the line out fixes it indeed. But as soon as the class gets transformed into native class syntax + native `get`ters this line should be back in the code. I don't want to leave it like is, but postpone the transformation, since it will be included in future PR. Or do you see a better approach w/o refactoring to native classs+getters?

Reply:
No since its addressed anyway in the next few pr's i think its ok.

Reply:
No since its addressed anyway in the next few prs i think its ok.

Reply:
I tested it for a little, and deleting the `this._activitySum();` call, which is a side effect in the `activitySum` getter, fixed it. There is still some odd timing issues (wrong sum gets displayed for half a second when changing the focused task), but those will be fixed with the native class transformation. 

Could you please approve again @velrest :-)